book/src/configuration.md

```diff
@@ -179,7 +179,7 @@ All git related options are only enabled in a git repository.
 |`git-ignore` | Enables reading `.gitignore` files | `true`
 |`git-global` | Enables reading global `.gitignore`, whose path is specified in git's config: `core.excludesfile` option | `true`
 |`git-exclude` | Enables reading `.git/info/exclude` files | `true`
-|`max-depth` | Set with an integer value for maximum depth to recurse | Defaults to `None`.
+|`max-depth` | Set with an integer value for maximum depth to recurse | `none`.
```

Comment:
`none` wouldn't be a valid value for this. Maybe we can say something like "Unset by default" to be clearer